<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border-image: url(:/pictures/background.png);}</string>
  </property>
  <widget class="QLabel" name="Name_Frog">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;华文中宋&quot;;</string>
   </property>
   <property name="text">
    <string>请为您的小青蛙取一个名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(174, 196, 200);
font: 9pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(174, 196, 200);
font: 9pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
